Bottom: 1203552e0323bbff82207498c9c26c1c3f05f56b
Top:    8b58e187aff8156ed1d16d6af530bd0bffab5561
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-24 10:38:59 +0000

Remove unneeded Undo serial number save / restore in netlist lookup

The RatFindHook function does not call anything which will increment
the Undo serial number, so the save / restore calls are not required.


---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index 96315b4..addb0d0 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -518,11 +518,9 @@ netlist_select_cb (GtkWidget * widget, gpointer data)
   InitConnectionLookup ();
   ResetConnections (false);
 
-  SaveUndoSerialNumber ();
   for (i = selected_net->EntryN, entry = selected_net->Entry; i; i--, entry++)
     if (SeekPad (entry, &conn, false))
       RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2, true, true);
-  RestoreUndoSerialNumber ();
 
   SelectConnection (select_flag);
   ResetConnections (false);
diff --git a/src/hid/lesstif/netlist.c b/src/hid/lesstif/netlist.c
index bf48b09..a2d3507 100644
--- a/src/hid/lesstif/netlist.c
+++ b/src/hid/lesstif/netlist.c
@@ -138,11 +138,9 @@ nbcb_select_common (LibraryMenuTypePtr net, int pos, int select_flag)
   InitConnectionLookup ();
   ResetConnections (false);
 
-  SaveUndoSerialNumber ();
   for (i = net->EntryN, entry = net->Entry; i; i--, entry++)
     if (SeekPad (entry, &conn, false))
       RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2, true, true);
-  RestoreUndoSerialNumber ();
 
   SelectConnection (select_flag);
   ResetConncetions (false);
